src/Microsoft.ML.Transforms/Text/CharTokenizeTransform.cs

```diff
@@ -119,6 +121,7 @@ private CharTokenizeTransform(IHost host, ModelLoadContext ctx, IDataView input)
             // <base>
             // byte: _useMarkerChars value.
             _useMarkerChars = ctx.Reader.ReadBoolByte();
+            CurrentModelVersion = ctx.Header.ModelVerReadable;
```

Comment:
>CurrentModelVersion [](start = 12, length = 19)

Oh I see what you've done here.. this is not good, static variables are essentially global state. They are not particular to an instance.